deploy/charts/venafi-kubernetes-agent/values.yaml

```diff
@@ -192,6 +192,19 @@ config:
     name:
     key:
 
+  # -- Configuration section for the Venafi Kubernetes Agent itself
+  venafiConnection:
+    # -- When set to false, the rendered output does not contain the
+    # VenafiConnection CRDs and RBAC. This is useful for when the
+    # Venafi Connection resources are already installed separately.
+    enabled: false
```

Comment:
People will probably want to install the CRDs separately but still use the venafi connection auth method. It looks like the helm chart doesn't allow you to do that at the moment, WDYT?

Reply:
Atanas also asked whether we could enable the venafi connection auth by default (i.e., `--venafi-connection` is used by default, but maybe not the CRDs). 

For example, imagine that a customer install Venafi Kubernetes Agent through the VMG/operator, the Helm installation should work regardless of whether they forgot to configure Venafi Kubernetes Agent. They will see in the logs that the last step is to create a venafi connection object.

Would that work?

Reply:
Partly done. I rearranged the Helm values. 
There are now

* `venafiConnection.include`: Like VEI this controls whether the VenafiConnection CRD and related service account and RBAC get installed. (default `false`).
* `config.venafiConnection.enabled`: This controls whether `--venafi-connection=venafi-components` is added to the command line and if true disables the old mounted Service account mechanism. (default `false`).